analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -326,7 +326,8 @@ open class Npm(
         }
     }
 
-    private val graphBuilder = DependencyGraphBuilder(NpmDependencyHandler(npmRegistry))
+    private val graphBuilder: DependencyGraphBuilder<NpmModuleInfo> =
+        DependencyGraphBuilder(NpmDependencyHandler(npmRegistry))
```

Comment:
Just for my understanding: Are you after making the `DependencyGraphBuilder` type explicit, or the template argument `NpmModuleInfo`? I would understand the latter somewhat, but for the first "main" type having something like

    val ... : Type = Type()

seem overly verbose to make `Type` explicit, as there is a immediate duplication of the `Type`.

Reply:
I'm after making the `<NpmModuleInfo` obvious.